mlflow/tracing/processor/inference_table.py

```diff
@@ -74,6 +75,8 @@ def on_start(self, span: OTelSpan, parent_context: Optional[Context] = None):
                 return
         span.set_attribute(SpanAttributeKey.REQUEST_ID, json.dumps(request_id))
         tags = {TRACE_SCHEMA_VERSION_KEY: str(TRACE_SCHEMA_VERSION)}
+        if depedencies_schema := maybe_get_dependencies_schema():
+            tags.update(depedencies_schema)
```

Comment:
Is this code needed for even langchain to write to inference table?

Reply:
Actually this is not necessary for langchain because we use MlflowLangchainTracer to pass tags into client.start_trace and inside this function we update the saved trace's tags. While for pyfunc model that's decorated with @mlflow.trace, we don't go through mlflowClient, so we need to fetch the tags inside processor directly. This sounds complex and redundant sometime to me, we'll see if we can improve afterwards 😆